.github/workflows/build.yaml

```diff
@@ -13,29 +17,33 @@ jobs:
     runs-on: ${{ matrix.archmap[matrix.arch] }}
     strategy:
       matrix:
-        app: [pilot, keeper, nexus]
+        app: [pilot, keeper, nexus, demo, bootstrap]
```

Comment:
Adding demo and bootstrap to the build matrix.

Demo can be used as an image to test SDK integration in e2e tests;
And Bootstrap is the new feature where we bootstrap SPIKE Nexus out-of-band to allow future HA capabilities.